radio/src/targets/horus/adc_driver.cpp

```diff
@@ -133,11 +133,11 @@ void adcInit()
 #define SPI_S1                         4
 #define SPI_6POS                       5
 #define SPI_S2                         6
-#define SPI_S3                         7
-#define SPI_S4                         8
+#define SPI_LS                         7

```

Comment:
I have `L1` on the left linear slider (left of the power button) and `L2` label on the right one. I see that old photos had S3 and S4 there. I did not know that.